assets/stylesheets/pages/_landing.scss

```
@@ -1,40 +1,50 @@
 .preamble {
-  font-size: 28px;
+  font-size: 38px;
   line-height: 1.4;
-  margin-bottom: 30px;
+  width: 42%;
```

```suggestion
  width: 50%;
```

See rationale on the `.featured-snippet` comment.